cyences_app_for_splunk/default/macros.conf

```diff
@@ -113,6 +113,27 @@ iseval = 0
 definition = `cs_tenable` sourcetype="tenable:io:vuln"
 iseval = 0
 
```

Comment:
On top here add the below comment:
# ================================== #
# Do not update these macros - Internal macros used to avoid known false positives  #
# ================================== #

Reply:
Add # before everyline, githhub took that as a formatting.